src/sdk-reference/js/6/essentials/error-handling/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: sdk.html.hbs
+algolia: true
+title: Error Handling
+description: How to handle errors with the SDK
+order: 100
+---
+
+# Error Handling
+
+Each of the SDK methods is likely to fail and return an error.  
```

Comment:
We should not have to tell Javascript developers how to handle a promise rejection.

Proposal:

```
All SDK methods return a promise, that can be rejected with a `KuzzleError` value in case of failure.

`KuzzleError` objects inherit the standard `Error` object, and add the following properties to it:


| Property   | Type    | Description                       |
| ---------- | ------- | --------------------------------- |
| `status` | Number | Status following [HTTP Standards](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) |
| `stack` | String | Error stacktrace (Only in development mode) |
```